res/values-de/strings.xml

```diff
@@ -19,8 +19,7 @@
   </string-array>
   <!--TODO re-enable when server-side folder size calculation is available   
     	<item>Biggest - Smallest</item>-->
-  <!--TODO re-enable when "Accounts" is available in Navigation Drawer-->
-  <!--<string name="drawer_item_accounts">Accounts</string>-->
+  <string name="drawer_item_accounts">Accounts</string>
```

Comment:
We can't include changes in translated languages through git, or will break daily merge of translations. Please, undo this change, let the equivalent in res/values/strings.xml.